cosmwasm/Makefile

```diff
@@ -54,21 +49,16 @@ tools/node_modules: tools/package-lock.json
 	cd tools && npm ci
 
 LocalTerra:
-	mkdir LocalTerra && \
-	cd LocalTerra && \
-	git init && \
-	git remote add origin  https://www.github.com/terra-money/LocalTerra.git && \
-	git fetch --depth 1 origin 958ff795f261f5ff2efc7b56604e2434eb76f7c4 && \
-	git checkout FETCH_HEAD
+	git clone --depth 1 https://www.github.com/terra-money/LocalTerra
```

Comment:
We're not going to anchor to a particular version?

Reply:
Oh, is this just a shallow copy of what the other Terra is using?  Then nevermind.

Reply:
Fair callout, this is what the original code was. We needed to pin the commit on the other once the terra 2 / classic split happened. This is for terra 2 so it can go back to pulling the latest to keep us up to date.